README.md

```diff
@@ -2,17 +2,22 @@
 
 The new home of the betanextbest.wellcomecollection.org website.
 
-# client
-
-    cd client
-    npm install
-    npm run compile OR npm run compile:watch (for developing)
-       
-# server
-    
-    cd server
-    npm install
-    npm run watch
-    
+## client
+
+    - `cd client`
```

Comment:
You'd still run into trouble once you get to step three: - `npm run compile` OR `npm run compile:watch` (for developing).

I think I favour a list of things being marked up as a list. But I don't care too much.